kolibri/plugins/learn/assets/src/views/ExamPage/index.vue

```diff
@@ -349,11 +349,7 @@
     },
     watch: {
       attemptLogItemValue(newVal, oldVal) {
-        // HACK: manually dismiss the perseus renderer message when moving
-        // to a different item (fixes #3853)
-        if (newVal !== oldVal && this.$refs.contentRenderer) {
-          this.$refs.contentRenderer.$refs.contentView.dismissMessage &&
-            this.$refs.contentRenderer.$refs.contentView.dismissMessage();
```

Comment:
how can all of this get deleted? is it because the logic about the content renderer existing is now elsewhere?

Reply:
This was implemented a long time ago in the perseus renderer, so this is old, superfluous code.

See here where it's dismissed when rendered https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue#L279 (also lots of other calls to `dismissMessage` when other user interaction has happened).